### PR TITLE
Update readme, depreciate -Pu and -Pn

### PR DESCRIPTION
**Pull request by Morganamilo**

### CODE REVIEW

File: doc/yay.8 [hunk 1]
```diff
@@ -118,8 +118,7 @@ Print current yay configuration\&.
 .PP
 \fB\-n \-\-numberupgrades\fR
 .RS 4
-Print number of packages that need to be updated\&. Note this does not perform
-a database refresh\&. Run \fByay \-Sy\fR Before this for an up to date result\&.
+Depreciated, use \fByay -Qu\fR and \fBwl -l\fR instead\%.
```
Comment (StephenBrown2) [inline doc/yay.8]: sp: Deprecated

File: doc/yay.8 [hunk 2]
```diff
@@ -131,8 +130,7 @@ be displayed\&.
 .PP
 \fB\-u \-\-upgrades\fR
 .RS 4
-Print Names of packages that need to be updated\&. Note this does not perform
-a database refresh\&. Run \fByay \-Sy\fR Before this for an up to date result\&.
+Depreciated, use \fByay -Qu\fR instead\%.
```
Comment (StephenBrown2) [inline doc/yay.8]: sp: Deprecated

File: doc/yay.8 (hunk 1 above)
Comment (Morganamilo) [reply doc/yay.8]: Annnddddd error 500 when trying to make a PR:

image

Comment (StephenBrown2) [reply doc/yay.8]: Ahhh rats! Just a few minutes too late.

Lovely error message that. :-o

**Pull request merged**